src/utils/localStorage.ts

```diff
@@ -8,3 +16,27 @@ export const getLocalStorageToken = (venueId: string) =>
 
 export const removeLocalStorageToken = (venueId: string) =>
   localStorage.removeItem(accessTokenKey(venueId));
+
+export const storeCustomLoaders = (customLoaders: CustomLoader[]) =>
+  localStorage.setItem(
+    LocalStorageKeys.customLoaders,
+    JSON.stringify(customLoaders)
+  );
+
+export const retrieveCustomLoaders = (): CustomLoader[] => {
+  const customLoadersJson =
+    localStorage.getItem(LocalStorageKeys.customLoaders) ??
+    JSON_STRINGIFIED_ARRAY;
```

Comment:
[TODO] Should we change this to use `sessionStorage` instead of `localStorage`? That way we won't have the chance to cache long-lasting bugs that persist even between browser close/reopens.. which might save us from future pain.

Reply:
fixed in [`8f6191b` (#1240)](https://github.com/sparkletown/sparkle/pull/1240/commits/8f6191b98377b3c30837d729cae767cde7d9cf94)